src/components/TranslationBannerLegal.tsx

```diff
@@ -125,20 +42,74 @@ const TranslationBannerLegal: React.FC<IProps> = ({
   }, [originalPagePath, shouldShow])
 
   return (
-    <BannerContainer isOpen={isOpen}>
-      <StyledBanner>
-        <BannerContent isPageRightToLeft={isPageRightToLeft}>
-          <Row>
-            <H3>
+    <Box
+      position="fixed"
+      display={isOpen ? "block" : "none"}
+      bottom={{ base: 0, md: 8 }}
+      right={{ base: 0, md: 8 }}
+      zIndex="99"
+    >
+      <Flex
+        bg="infoBanner"
+        color="black300"
+        justify="space-between"
+        maxW={{ base: "full", md: "600px" }}
+        maxH="full"
+        p={4}
+        borderRadius="2px"
+        boxShadow={{
+          base: `-4px 10px 0px ${text} 10%`,
+          md: "rgba(0, 0, 0, 0.16) 0px 2px 4px 0px",
+        }}
+      >
+        <Flex
+          direction="column"
+          align={isPageRightToLeft ? "flex-end" : "flex-start"}
+          m={4}
+          mt={{ base: 10, sm: 4 }}
+        >
+          <Flex
+            align={{ base: "flex-start", sm: "center" }}
+            flexDirection={{ base: "column-reverse", sm: "row" }}
+            mb={4}
+          >
+            <Heading
+              as="h3"
+              fontSize={{ base: "1.25rem", md: "1.5rem" }}
+              fontWeight="bold"
+              lineHeight="100%"
+              my={0}
+            >
               <Translation id="translation-banner-no-bugs-title" />
-              <StyledEmoji ml={"0.5rem"} size={1.5} text=":bug:" />
-            </H3>
-          </Row>
-          <p>
+              <Emoji
+                text=":bug:"
+                fontSize="3xl"
+                pt={2}
+                ml={2}
+                mb={{ base: 4, sm: "auto" }}
+              />
+            </Heading>
+          </Flex>
+          <Text>
             <Translation id="translation-banner-no-bugs-content" />
-          </p>
-          <ButtonRow>
-            <ButtonPrimary
+          </Text>
+          <Flex
+            align={{ base: "flex-start", sm: "center" }}
+            flexDirection={{ base: "column", sm: "row" }}
+          >
+            <Button
+              py={2}
+              px={3}
+              bgColor="primary"
+              color="buttonColor"
+              border="1px solid transparent"
+              _hover={{
+                bgColor: "primaryHover",
+                boxShadow: cardBoxShadow,
+              }}
+              _active={{
+                bgColor: "primaryActive",
+              }}
```

Comment:
You can remove these styles. The default Button is going to render the `solid` variant with the correct color scheme. You can see the details [here](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/components/Button.ts).

```suggestion
            <Button
```